packages/core/src/lib/TwaManifest.ts

```diff
@@ -201,6 +207,10 @@ export class TwaManifest {
     this.enableSiteSettingsShortcut = data.enableSiteSettingsShortcut != undefined ?
       data.enableSiteSettingsShortcut : true;
     this.isChromeOSOnly = data.isChromeOSOnly != undefined ? data.isChromeOSOnly : false;
+    this.isMetaQuest = data.isMetaQuest != undefined ? data.isMetaQuest : false;
+    this.scope = data.scope;
```

Comment:
@andreban Should I add the `scope` at all? It is not actually used in TWA and is only needed to generate `fullScopeUrl` that is used only by Meta Quest.

Reply:
`merge` would need to reflect adding / removing / changing a scope?